src/components/post-forms.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import "../App.css";
+import PostForm from "./post.form";
+
+class PostForms extends Component {
+  render() {
+    const { tasks, onAdd, onDelete } = this.props;
+    return (
+      <div className="post-forms-container">
+        {tasks.map((task) => (
```

Comment:
if you change this line to `{tasks.map((task, idx) => (` than, you may replace `tasks.indexOf(task)` into `idx`